TriggersTemplate.gs

```diff
@@ -0,0 +1,72 @@
+/**
+ * The event handler triggered when the attached spreadsheet is opened.
+ * @param {Event} e The onOpen event object.
+ * e.authMode A value from the ScriptApp.AuthMode enum.
+ * e.source A Spreadsheet object, representing the Google Sheets file to which the script is bound.
+ * e.triggerUid ID of trigger that produced this event (installable triggers only).
+ * e.user User object representing the active user, if available.
+ * @return {SpreadsheetTriggerBuilder}  Builder for spreadsheet trigger chaining.
+ */
+function onOpen(e){
+}
+
+/**
+ * The event handler triggered when an edit is made to the attached spreadsheet.
+ * @param {Event} e The onEdit event object.
+ * e.authMode A value from the ScriptApp.AuthMode enum.
```

Comment:
Indent this. and the lines below by 2 spaces to the right, also in other methods